test/e2e/page-objects/pages/home/homepage.ts

```diff
@@ -296,6 +299,19 @@ class HomePage {
     );
   }
 
+  /**
+   * Checks if the balance empty state is displayed on homepage.
+   * Criteria:
+   * - The account group has a zero balance across all aggregated mainnet networks.
+   * - The account group is not on a test network
+   * - The account group is not in a cached state
+   * Not a replacement for checkExpectedBalanceIsDisplayed('0') this is still valid in certain cases.
+   */
+  async checkBalanceEmptyStateIsDisplayed(): Promise<void> {
+    console.log('Check balance empty state is displayed on homepage');
+    await this.driver.waitForSelector(this.emptyBalance);
+  }
```

Comment:
Added a new method to the homepage page object to check for the balance empty state component instead of relying on a 0 balance check. Not sure how many e2e tests are run with the `isMultichainAccountsState2Enabled` feature flag enabled, but so far it looks like only `test/e2e/dist/vault-decryption-chrome.spec.ts` is the only one. All other e2e tests are passing.